packages/libs/react-components/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,77 @@
+import { SystemIcons } from '../Icons';
+
+import {
+  AbsoluteButton,
+  StyledButton,
+  StyledHeading,
+  StyledIconButton,
+  StyledNotification,
+  StyledText,
+} from './styles';
+
+import React, { FC } from 'react';
+
+export interface INotificationProps {
+  icon?: (typeof SystemIcons)[keyof typeof SystemIcons];
+
+  title: string;
+  description?: string;
+  displayCloseButton?: boolean;
+  expand?: boolean;
+
+  actionButtonLabel?: string;
+  actionButtonOnClick?: () => void;
+  displayActionButton?: boolean;
+
+  color?: string;
+
+  simplified?: boolean;
+}
+
+export const Notification: FC<INotificationProps> = ({
+  icon,
+  title,
+  description,
+  displayCloseButton = false,
+  actionButtonLabel,
+  actionButtonOnClick,
+  displayActionButton = false,
+  color = 'default',
+  simplified = false,
+  ...props
```

Comment:
We want to keep this library strict, so we usually want to avoid exposing the ability to pass anything to components. What other props do we want to pass on through this spread operator?